.github/workflows/e2e-external.yml

```diff
@@ -0,0 +1,36 @@
+name: E2E over minikube (External Contribution)
+
+on:
+  pull_request_target:
+
+permissions:
+  contents: read
+
+jobs:
+  e2e-tests-external:
+    name: Run E2E Tests
+    # Do not run e2e tests if PR has skip-e2e label or if it's a non-forked repo (internal contribution)
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'ci/skip-e2e') && github.event.pull_request.head.repo.full_name != 'newrelic/k8s-agents-operator' }}
+    runs-on: ubuntu-latest
+    environment: E2E # Required so that approval is required to run this job since it is untrusted (forked).
+    strategy:
+      max-parallel: 6 # len(k8sVersion)/2 is a good number to have here
+      matrix:
+        # Latest patch version can be found in https://github.com/kubernetes/website/blob/main/content/en/releases/patch-releases.md
+        # Some versions might not be available yet in https://storage.googleapis.com/kubernetes-release/release/v1.X.Y/bin/linux/amd64/kubelet
+        k8sVersion: [ "v1.34.1", "v1.33.5", "v1.32.9", "v1.31.13", "v1.30.14"]
+    steps:
+      - name: Checkout GitHub Repository
+        uses: actions/checkout@8e8c483db84b4bee98b60c0593521ed34d9990e8 # v6.0.1
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Download minikube
+        run: |
+          curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+          sudo install minikube-linux-amd64 /usr/local/bin/minikube && rm minikube-linux-amd64
+
+      - name: Run E2E tests
+        run: |
+          cd tests/e2e
+          ./e2e-tests.sh --k8s_version ${{ matrix.k8sVersion }} --license_key ${{ secrets.K8S_AGENTS_E2E_LICENSE_KEY }} --run_tests
```

Comment:
Since we're exposing secrets.K8S_AGENTS_E2E_LICENSE_KEY to this test suite. 
Would that mean that a bad actor could edit one of the tests to print this value?